index.js

```diff
@@ -52,6 +53,6 @@ const applyOutputDefaults = (output) => ({
   js: output.js || 'bundle.js'
 });
 
-const resolveAbsolutePath = (relativePath) => path.resolve(process.cwd(), relativePath);
+const resolveAbsolutePath = (relativePath) => resolveFilePathIfExists(process.cwd(), relativePath);
```

Comment:
Looks like this function belongs in `utils` now.

Reply:
done